src/App.tsx

```diff
@@ -15,49 +17,64 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+const App: React.FC = () => {
+  const [isListVisible, setisListVisible] = useState(false);
+  const [isReversed, setisReversed] = useState(false);
+  const [sortBy, setSortBy] = useState('');
+
+  const reset = () => {
+    setisReversed(false);
+    setSortBy('');
+  };
+
+  const visibleGoods = [...goodsFromServer];
```

Comment:
I think Oleksandr did mean to make smth like that:

And your function should be placed for example here `src -> helpers -> get-sorted-goods.ts`
```suggestion
  const visibleGoods = getSortedGoods(goodsFromServer, sortBy, isReversed);
```